### ISSUE TITLE
Track contributors with orbit?

### ISSUE THREAD

**Issue by mrocklin** [dask/community#126]
So, folks from [Orbit](https://orbit.love/) reached out recently.  They make a product designed to help people understand OSS engagement.  It looks like they track issues/comments/PRs/etc.. by individual and then give analytics around that. It seems to be more oriented towards for-profit companies as an extra feed to put into customer management software, but I'm actually a bit more curious about it from a community building / maintainer-growth standpoint.

Their demo was compelling enough that I'm tempted to try it out for a bit.  This would involve registering them as an app on the Dask org, which gives them a feed of public events.

There is a bit of a fine line to walk here for me in particular due to the non-profit/for-profit split.  I can promise not to share any specific user data with for-profit Coiled activities (we wouldn't want sales people cold-calling people who raise issues), but some amount of information leakage at a broad scale is probably hard to guarantee.

I'm also very happy to just not do this if it makes people feel uncomfortable.  This is more of a curiosity for me than anything else.

**Comment by jakirkham** [comment]
I'd want to know more about what they do with that data before doing this.

**Comment by martindurant** [comment]
Can we hear more about the potential benefits, and some sort of description of what data is potentially harvested?

**Comment by mrocklin** [comment]
> I'd want to know more about what they do with that data before doing this.
> Can we hear more about the potential benefits, and some sort of description of what data is potentially harvested?

I don't know much beyond what is on their site.  The objective of this experiment would be to learn more.  That being said, github.com/dask doesn't have any private data, so they're not getting anything that an active scraper couldn't get.

**Comment by quasiben** [comment]
I would be up for this experiment.  I would hope to see a view of users and what they are doing / want / need / etc.  If they share with just coiled would this info bbe fully shared with the larger community ?

**Comment by jakirkham** [comment]
> > I'd want to know more about what they do with that data before doing this.
> > Can we hear more about the potential benefits, and some sort of description of what data is potentially harvested?
> 
> I don't know much beyond what is on their site. The objective of this experiment would be to learn more. That being said, github.com/dask doesn't have any private data, so they're not getting anything that an active scraper couldn't get.

Do they need to install a webhook in the org? Do users grant permission to it? Does it comply with GDPR and CCPA?

**Comment by mrocklin** [comment]
Unfortunately I don't know the answers to your questions

**Comment by mrocklin** [comment]
> I would be up for this experiment. I would hope to see a view of users and what they are doing / want / need / etc. If they share with just coiled would this info bbe fully shared with the larger community ?

I agree that it makes sense to expose this to a few folks.  To be clear, I don't think think that *coiled* is doing this.  I'm trying to do this with my Dask hat / community building hat on.  If this works well my hope would be to hire a community manager role within the Dask org that would try to take on my current role of connecting people up.

**Comment by mrocklin** [comment]
I'm also only planning to look at this stuff as an experiment.  I'm not proposing marrying ourselves to it.  I think that a lot of these larger questions could be deferred after we have more information.

**Comment by mrocklin** [comment]
If there are no objections I would like to go further with this.  This came up when planning the Dask Summit and trying to attract proposals.  It would be nice to see who is around from whom we could solicit proposals.

I can't tell if the comments raised here are mostly expressiing curiosity, or are intented to block progress.  I'm going to assume the former unless folks speak up.

**Comment by TomAugspurger** [comment]
+1 from me.

**Comment by jakirkham** [comment]
Based on discussion at the monthly meeting, it sounds like we will try things out for a week. Then Matt will share some initial thoughts about the service, how it's going, and thoughts on whether to continue using. Does that sounds like an accurate summary to folks here?

**Comment by jrbourbeau** [comment]
That sounds like an accurate summary to me
